data/core/dirwatch.lua

```diff
@@ -171,10 +171,10 @@ end
 
 
 -- compute a file's info entry completed with "filename" to be used
--- in project scan or falsy if it shouldn't appear in the list.
+-- in project scan or return false if it shouldn't appear in the list.
 local function get_project_file_info(root, file, ignore_compiled)
   local info = system.get_file_info(root .. PATHSEP .. file)
-  -- info can be not nil but info.type may be nil if is neither a file neither
+  -- info can not be nil but info.type may be nil if it is neither a file nor
```

Comment:
I think this changes the meaning of the comment.

Reply:
oh right.

Reply:
changed to 'may not be'

Reply:
hmm, it's hard to detect whether the comment is stating facts or what is being let through :/, that's where my confusion came from.

Reply:
OK, I like the newest wording. [252379](https://github.com/lite-xl/lite-xl/pull/1549/commits/2523279f23a815c2c1a101e863ca1bd5b646f851)